services/tilesets.js

```diff
@@ -16,6 +17,11 @@ var Tilesets = {};
  *
  * @param {Object} [config]
  * @param {string} [config.ownerId]
+ * @param {'raster'|'vector'} [config.type] - Filter results by tileset type, either `raster` or `vector`.
+ * @param {number} [config.limit=100] - The maximum number of tilesets to return, from 1 to 500. The default is 100.
```

Comment:
great now that the default is specified with JSDoc there's no need to include that in the description as you'll see in the generated docs/services.md it get's appended.